doc/user/content/free-trial-faqs.md

```diff
@@ -7,55 +7,75 @@ menu:
+free trial account so you can explore the product and start building! This page
+answers some frequently asked questions about free trials.
 
 ## What are the limits of a free trial?
 
-In Materialize, [clusters](/get-started/key-concepts/#clusters) are the pools of compute resources for running your workloads. The size and replication factor of each cluster determines its [credit usage](/sql/create-cluster/#credit-usage).
+In Materialize, [clusters](/get-started/key-concepts/#clusters) are the pools of
+compute resources for running your workloads. The size and replication factor
+of each cluster determines its [credit usage](/sql/create-cluster/#credit-usage).
 
-During your free trial, the credit consumption rate across all clusters in a region cannot exceed 4 credits per hour at any point in time. This limit should accommodate most trial scenarios.
+During your free trial, the credit consumption rate across all clusters in a
+region cannot exceed 4 credits per hour at any point in time. This limit should
+accommodate most trial scenarios.
 
 For example, let's say you have 3 clusters in a region:
 
 Cluster     | Size      | Replication factor | Credits per hour
 ------------|-----------|--------------------|-----------------
 `ingest`    | `50cc`    | 1                  | 0.5
 `compute`   | `50cc`    | 2                  | 1 (0.5 each)
-`quickstart`| `50cc`    | 1                  | 0.5
+`quickstart`| `100cc`   | 1                  | 1
 
-In this case, your credit consumption rate would be 2 credits per hour, which is under the rate limit of 4 credits per hour.
+In this case, your credit consumption rate would be 2.5 credits per hour, which
+is under the rate limit of 4 credits per hour.
 
 ## How long does a free trial last?
 
 14 days.
 
-If you need additional time, please [chat with our team](http://materialize.com/convert-account/). Otherwise, Materialize will delete your resources and data at the end of the trial period.
+If you need additional time, please [chat with our team](http://materialize.com/convert-account/).
+Otherwise, Materialize will delete your resources and data at the end of the
+trial period.
 
 ## How do I monitor my credit consumption rate?
 
-To see your current credit consumption rate, measured in credits per hour, run the following query:
+To see your current credit consumption rate, measured in credits per hour, run
+the following query against Materialize:
 
 ```sql
 SELECT sum(s.credits_per_hour) AS credit_consumption_rate
   FROM mz_cluster_replicas r
   JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;
 ```
 
-For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming credits at a rate of `3.5` per hour:
+For example, if you start your free trial by following the [getting started guide](/get-started/quickstart),
+or if you otherwise only use the pre-installed `quickstart` cluster
+(`100cc`), you will end up consuming `1` credit per hour:
```

Comment:
```suggestion
(`25cc`), you will end up consuming `1` credit per hour:
```

(quickstarts will be `25cc` starting very soon)

Reply:
I'll adjust things then!

Reply:
Er, sorry, as in probably later today: https://github.com/MaterializeInc/cloud/pull/9202

Reply:
:le-sigh:

Reply:
I'll patch in my fnlist PR.